sky/packages/updater/pubspec.yaml

```diff
@@ -1,9 +1,16 @@
-name: sky_updater
+name: updater
 version: 0.0.1
 author: Chromium Authors <sky-dev@googlegroups.com>
 description: The autoupdater for flutter
 homepage: http://flutter.io
 dependencies:
-  mojo: ^0.0.21
+  mojo: '>=0.1.0 <0.2.0'
+  sky: any
+  sky_services: any
+dependency_overrides:
+  sky:
+    path: ../sky
+  sky_services:
+    path: ../sky_services
```

Comment:
I have to admit, I arrived at this through a bit of random trial and error. I don't really understand the dart package dependency system.

If I understand what you're saying, I need to split this change in 2 parts:
1. Update activity.mojom (and add update_service.mojom)
2. The rest of the change, that depends on those interface changes.

Is that right?